internal/core/renderer.rs

```diff
@@ -116,4 +117,10 @@ pub trait RendererSealed {
     fn resize(&self, _size: crate::api::PhysicalSize) -> Result<(), PlatformError> {
         Ok(())
     }
+
+    /// Re-implement this function to support Window::grab_window(), i.e. return
```

Comment:
this comment seems outdated since there isn't a grab_window funtion anymore

Reply:
That’s still the name in the public API.